answer_rocket/graphql/schema.py

```diff
@@ -234,6 +234,23 @@ class CopilotSkillRunResponse(sgqlc.types.Type):
     data = sgqlc.types.Field(JSON, graphql_name='data')
 
 
+class CopilotTopic(sgqlc.types.Type):
```

Comment:
If we end up renaming this to `MaxCopilotTopic` in the Max PR, then this will need to get regenerated.